### DISCUSSION TITLE
Google Login no va a funcionar en Docker

### DISCUSSION THREAD

**Discussion by MarioArocaPaez** [perealberto/decide-single-zambrano#130] [Show and tell]
Buenas chicos escribo por aquí para comentaros el tema de que el Google Login no va a funcionar en Docker. El problema que hay con esto es que para que vaya el Google Login hay que añadir la URL que solicita el login al google cloud platform. En nuestro caso para nuestro docker sería esta:
![imagen](https://github.com/3ralon/decide-single-zambrano/assets/73229438/3e584095-dbed-4b8f-b346-f51bcb8e0cb9)
El problema es que con esta configuración actual, desde docker puedes hacer la solicitud para el google login sin problema, la cosa es que justo después de entrar tus credenciales y autenticarte la pagina se quedará asi:
![imagen](https://github.com/3ralon/decide-single-zambrano/assets/73229438/57b2a575-e4ca-4898-99a9-92b4cdbc55c1)
He estado investigando y esto es porque a Google no le gusta esta URL porque no la considera segura y no hay forma de cambiarlo a menos que registremos esta URL. 
Otra solución que me propuso el profesor fue cambiar el archivo hosts para que cuando se haga esta solicitud se haga desde el puerto 8000 que es el que funciona perfectamente lo he estado probando:
![imagen](https://github.com/3ralon/decide-single-zambrano/assets/73229438/0e105757-7fd4-4329-8eae-1e7ee4d953f0)
Y el problema es que cuando nosotros hacemos esto nosotros vemos en la url de nuestro navegador que se está haciendo por el puerto 8000 pero en verdad la solicitud desde dentro se está haciendo desde la misma dirección de Docker que no le gusta a google por lo que volvemos al mismo problema de antes.
Ya de aquí no soy capaz de encontrar otra solución, creo que está completamente fuera de nuestro control esto. Os pido disculpas porque realmente creia que podria hacer esto.

**Comment by claugp06** [comment]
No te preocupes. Se ha intentando arreglar y no funciona. Por lo menos sabemos a que se debe así que buen trabajo. Por mi parte, lo dejaría como esta y que quede aquí registrado la decisión tomada
